src/core/ddsi/src/q_init.c

```diff
@@ -1403,7 +1403,7 @@ int rtps_start (struct q_globals *gv)
   }
   if (gv->listener)
   {
-    create_thread (&gv->listen_ts, gv, "listen", (uint32_t (*) (void *)) listen_thread, gv->listener);
+    (void) create_thread (&gv->listen_ts, gv, "listen", (uint32_t (*) (void *)) listen_thread, gv->listener);
```

Comment:
Again, casting away an error here is not right. And there is even a FIXME right there!

Reply:
I will check the return code and log a message.